### DISCUSSION TITLE
Tokens Studio Pro

### DISCUSSION THREAD

**Discussion by robomatic** [primefaces/community#561] [Figma UI Kit] [answered]
To use the figma UI kit to it's full potential do you require Tokens Studio Pro or will Tokens Studio free be able to make all necessary changes. What can be changed with the free version of Tokens Studio Pro and the UI kit?

**Comment by onursenture** [comment, marked as answer]
The free version of Tokens Studio is sufficient to work with tokens.

We try to refrain from using features like the Modifier from Tokens Studio Pro to enable everyone to use the free version to its full potential.

If you do not plan to use things like branch switch to improve your design organization or extend the UI Kit with features like multi-dimensional themes, staying on the Free plan will not make much of a difference.
